src/components/Accordion/Accordion.stories.mdx

```diff
@@ -331,3 +336,11 @@ export const faqContent = [
 
 <Accordion accordionData={faqContent} />
 ```
+
+## Example with panelMaxHeight Prop
```

Comment:
Can you add one more example that includes a few paragraphs of text in the accordion panel?